### DISCUSSION TITLE
Como puedo obtener el nombre de una tasks, proveniente de un pbupdatetask?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#233] [2. Questions and Answers] [answered]
Hola no sé si está ben explicado pero dejo el detalle (si me pueden cambiar el nombre de la discusión se agradecería).

Tengo el siguiente orden de un bot:

1.- Crear tarea (create_task)
2.- copiar mensajes al canal de la tarea (copy_survey)
3.- Upgradear tarea (para colocar el serial) (update_task)
4.- Enviar email (send_email)

(nombre de las rutinas)

Para el caso de enviar el email con el nuevo nombre de la task, está correcto la expresión: $OUTPUT#update_task#task|name o sería $OUTPUT#update_task#data|name?.

Lo pregunto porque nunca he hecho tomar el nombre de la task por un update_task.

Gracias :>

**Comment by guillevalin** [comment, marked as answer]
@NicolasPLG Según la definición del bot PBUpdateTask, en el output manda un objeto con el parámetro "task" que contiene la tarea, por lo tanto, lo válido sería: ```$OUTPUT#update_task#task|name```.

**Reply by NicolasPLG** [reply]
perfecto gracias @guillevalin
